man/adoc/bpftrace.adoc

```diff
@@ -3443,15 +3443,12 @@ system.
 
 === max_strlen
 
-Default: 64
+Default: 1024
 
-Number of bytes allocated on the BPF stack for the string returned by str().
-Make this larger if you wish to read bigger strings with str().
-Beware that the BPF stack is small (512 bytes), and that you pay the toll again inside printf() (whilst
-it composes a perf event output buffer).
-So in practice you can only grow this to about 200 bytes.
+The maximum length (in bytes) for values created by `str()`, `buf()` and `path()`.
 
-Support for even larger strings is being discussed: https://github.com/bpftrace/bpftrace/issues/305.
+This limit is necessary in order to size buffers data must be staged in before they can be interacted with.
```

Comment:
Sorry, now you got me in editing mode. I won't block but how about
```suggestion
This limit is necessary because BPF requires the size of all program strings to be declared up front. This is the size for all string variables in bpftrace unless specified at the call site.
```

Reply:
Edited a little to account for the other types, but overall taken